lib/util.ml

```diff
@@ -1267,110 +1297,71 @@ let string_of_witness_kind_raw witness_kind =
 let bpath bname = !GWPARAM.bpath bname
 
 (* ************************************************************************ *)
-(*  [Fonc] etc_file_name : config -> string -> string                       *)
-
+(*  [Fonc] etc_file_name : config -> string -> string                      *)
```

Comment:
My refactoring plan was to divide this function in three functions:
1. A function that generates all the searched directories. This function depends only on the configuration and the base mode (reorg or not). 
2. A function that searches the file in each directory in ascending order and return the first match.
3. A function equality passed to the previous one to handle the optional `.txt` extension.